airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json

```diff
@@ -16,8 +16,8 @@
       },
       "dataset_id": {
         "type": "string",
-        "description": "The BigQuery dataset id that will house replicated tables.",
-        "title": "Dataset ID"
+        "description": "Default BigQuery Dataset ID tables are replicated to if the source does not specify a schema.",
```

Comment:
Result: 
<img width="978" alt="Screen Shot 2021-03-27 at 1 38 45 AM" src="https://user-images.githubusercontent.com/9772859/113505365-eec9a400-9570-11eb-9cf3-88ae651bb8fc.png">

Reply:
i think this might be the wrong screenshot? i think this is the one for redshift?

Reply:
but reading the actual json, i think it's reasonable.

Reply:
oops. got it confused hehe

Reply:
for posterity, this is what this looks like (with the namespace change)

![image](https://user-images.githubusercontent.com/9772859/113654636-d79ec980-96ca-11eb-90fd-170ea7950a85.png)